### PR TITLE
Feature/hs 2768

### PR DESCRIPTION
**Pull request by slavovojacek**
@dmitrijs-balcers can you pls quickly review if I'd missed something?
@Gilbert09 can you pls implement the below HTML changes and deploy?

Many thanks!

### CODE REVIEW

File: js/utils/Logger/Logger.ts [hunk 1]
```diff
@@ -2,36 +2,28 @@ module hirespace {
     'use strict';
 
     export class Logger {
-        static log(data: any) {
-            if (hirespace.Debug.getEnvironment() !== 'development') {
-                return false;
-            }
+        static execute(type: string, data: any, notify?: boolean) {
```
Comment (dmitrijs-balcers) [inline js/utils/Logger/Logger.ts]: Shouldn't this method be private?

Comment (slavovojacek) [inline js/utils/Logger/Logger.ts]: it originally was but I would not be able to test it externally :( also, it does not change the scope in compiled JS anyway..

File: js/shared/Notification/Notification.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+module hirespace {
+    'use strict';
+
+    let clearNotificationTimeout;
+
+    export class Notification {
+        static generate(message: string, type: string) {
+            $('#notification').attr('class', '').addClass('active ' + type).html('<main>' + message + '</main>');
+
+            if (clearNotificationTimeout) clearTimeout(clearNotificationTimeout);
```
Comment (dmitrijs-balcers) [inline js/shared/Notification/Notification.ts:10]: alternative `clearNotificationTimeout && clearTimeout(clearNotificationTimeout)`

File: js/utils/Logger/Logger.ts (hunk 1 above)
Comment (slavovojacek) [inline js/utils/Logger/Logger.ts]: Actually you're right, I am not testing this one component anymore, just the 4 below so will change to private :))

Comment (dmitrijs-balcers) [inline js/utils/Logger/Logger.ts]: If you need to unit-test it, then you can extract it to the external utils class, and test it through it. In such way you will have a clean Logger class, and also testable "private method"

**Comment by dmitrijs-balcers** [comment]
Code looks good. Good Job

**Comment by slavovojacek** [comment]
Thanks, @Gilbert09 I assigned this to you, can you pls merge when u done implementing the layout changes? thanks

**Comment by Gilbert09** [comment]
All done - @slavomirvojacek

**Comment by slavovojacek** [comment]
thanks

**Pull request merged**